pkg/templates/templates_test.go

```diff
@@ -529,6 +539,95 @@ func TestResolveTemplate(t *testing.T) {
 	}
 }
 
+func TestReferencedObjs(t *testing.T) {
```

Comment:
This is missing a test for `lookup`.

Reply:
right, we have no  tests for lookup right now in other TemplateResolve tests as well, the test setup curently only for SimpleClient not dynamicClient ( dynamic resources)

Reply:
I see--thanks for clarifying!